tests/cmake/storage_cuda/main.cpp

```diff
@@ -0,0 +1,9 @@
+#include <gridtools/storage/builder.hpp>
+#include <gridtools/storage/cuda.hpp>
+
+int main() {
+    auto builder = gridtools::storage::builder<gridtools::storage::cuda>.type<int>().dimensions(2);
+    auto my_storage = builder();
+    my_storage->host_view()(0) = 10;
+    my_storage->get_target_ptr();
```

Comment:
This is enough to test CUDA runtime is working, right? It does a sync?!

Reply:
Strictly speaking this is not enough.
The proper test should take `get_target_ptr()` pointer; copy the memory back to the host using `cudaMemcpy`; cast to `int` and check that it is `10`

Reply:
Agree. I'll add it.

Reply:
done